### PR TITLE
copying blank ballot, cast ballot, ballot check to here

### PR DESCRIPTION
**Pull request by windoverwater**
Just a PR for the three data files

### CODE REVIEW

File: src/vtp/web/api/data/receipts.26.csv [hunk 1]
```diff
@@ -0,0 +1,101 @@
+"0000 - US president","0001 - US senate","0002 - governor","0003 - County Clerk","0004 - Question 1 - should the starting time of the annual town meeting be moved to 6:30PM?"
```
Comment (ion-oset) [inline src/vtp/web/api/data/receipts.26.csv:1]:
```suggestion
"0000 - U.S. President","0001 - U.S. senate","0002 - Governor","0003 - County Clerk","0004 - Question 1 - should the starting time of the annual town meeting be moved to 6:30 PM?"
```

File: src/vtp/web/api/data/cast-ballot.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                        "0: Jean-Luc Picard"
+                    ],
+                    "tally": "plurality",
+                    "uid": "0003"
+                }
+            }
+        ],
+        "GGOs/states/Massachusetts/GGOs/towns/Concord": [
+            {
+                "Question 1 - should the starting time of the annual town meeting be moved to 6:30PM?": {
```
Comment (ion-oset) [inline src/vtp/web/api/data/cast-ballot.json:128]:
```suggestion
                "Question 1 - should the starting time of the annual town meeting be moved to 6:30 PM?": {
```

File: src/vtp/web/api/data/blank-ballot.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                            "name": "Francis Foxtrot",
+                            "party": "Octagon Party"
+                        }
+                    ],
+                    "tally": "rcv",
+                    "uid": "0001"
+                }
+            },
+            {
+                "governor": {
```
Comment (ion-oset) [inline src/vtp/web/api/data/blank-ballot.json:80]:
```suggestion
                "Governor": {
```

File: src/vtp/web/api/data/cast-ballot.json [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        "GGOs/states/Massachusetts/GGOs/counties/Middlesex",
+        "GGOs/states/Massachusetts/GGOs/towns/Concord"
+    ],
+    "ballot_filename": "000,001,002,003,ballot.json",
+    "ballot_node": "GGOs/states/Massachusetts/GGOs/towns/Concord",
+    "ballot_subdir": "GGOs/states/Massachusetts/GGOs/towns/Concord",
+    "contests": {
+        "GGOs/states/Massachusetts": [
+            {
+                "US president": {
```
Comment (ion-oset) [inline src/vtp/web/api/data/cast-ballot.json:14]:
```suggestion
                "U.S. President": {
```

File: src/vtp/web/api/data/cast-ballot.json [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+                    "tally": "rcv",
+                    "ticket_offices": [
+                        "President",
+                        "Vice President"
+                    ],
+                    "uid": "0000"
+                }
+            },
+            {
+                "US senate": {
```
Comment (ion-oset) [inline src/vtp/web/api/data/cast-ballot.json:52]:
```suggestion
                "U.S. Senate": {
```

File: src/vtp/web/api/data/cast-ballot.json [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+                        "5: Francis Foxtrot",
+                        "3: David Delta",
+                        "1: Betty Beta"
+                    ],
+                    "tally": "rcv",
+                    "uid": "0001"
+                }
+            },
+            {
+                "governor": {
```
Comment (ion-oset) [inline src/vtp/web/api/data/cast-ballot.json:89]:
```suggestion
                "Governor": {
```

**Review by ion-oset — CHANGES_REQUESTED** [review]
Looks good to me. Mostly typographic changes.

File: src/vtp/web/api/data/blank-ballot.json [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+        "GGOs/states/Massachusetts/GGOs/counties/Middlesex",
+        "GGOs/states/Massachusetts/GGOs/towns/Concord"
+    ],
+    "ballot_filename": "000,001,002,003,ballot.json",
+    "ballot_node": "GGOs/states/Massachusetts/GGOs/towns/Concord",
+    "ballot_subdir": "GGOs/states/Massachusetts/GGOs/towns/Concord",
+    "contests": {
+        "GGOs/states/Massachusetts": [
+            {
+                "US president": {
```
[the remark this answers is not in the document]
Comment (windoverwater) [reply src/vtp/web/api/data/blank-ballot.json:14]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/blank-ballot.json [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+                    "tally": "rcv",
+                    "ticket_offices": [
+                        "President",
+                        "Vice President"
+                    ],
+                    "uid": "0000"
+                }
+            },
+            {
+                "US senate": {
```
[the remark this answers is not in the document]
Comment (windoverwater) [reply src/vtp/web/api/data/blank-ballot.json:48]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/cast-ballot.json [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+                            "ticket_names": [
+                                "Rey Skywalker",
+                                "Obi-Wan Kenobi"
+                            ]
+                        },
+                        {
+                            "name": "Square Party Ticket",
+                            "ticket_names": [
+                                "Atticus Finch",
+                                "Hermione  Granger"
```
[the remark this answers is not in the document]
Comment (windoverwater) [reply src/vtp/web/api/data/cast-ballot.json:27]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/blank-ballot.json [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+                    "choices": [
```
[the remark this answers is not in the document]
Comment (windoverwater) [reply src/vtp/web/api/data/blank-ballot.json:49]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/receipts.26.csv (hunk 1 above)
Comment (windoverwater) [reply src/vtp/web/api/data/receipts.26.csv:1]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/cast-ballot.json (hunk 2 above)
Comment (windoverwater) [reply src/vtp/web/api/data/cast-ballot.json:128]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/blank-ballot.json (hunk 3 above)
Comment (windoverwater) [reply src/vtp/web/api/data/blank-ballot.json:80]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/cast-ballot.json (hunk 4 above)
Comment (windoverwater) [reply src/vtp/web/api/data/cast-ballot.json:14]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/cast-ballot.json (hunk 5 above)
Comment (windoverwater) [reply src/vtp/web/api/data/cast-ballot.json:52]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

File: src/vtp/web/api/data/cast-ballot.json (hunk 6 above)
Comment (windoverwater) [reply src/vtp/web/api/data/cast-ballot.json:89]: https://github.com/TrustTheVote-Project/VTP-mock-election.US.14/commit/ccd386e05a84b85a6d671c5ff2dea65d9b613229

**Comment by windoverwater** [comment]
The merge of PR https://github.com/TrustTheVote-Project/VTP-web-api/pull/20 makes this OBE - closing.

**Pull request closed without merging**